app/src/main/res/layout/legend_row.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    >
+
+    <ImageView
+        android:id="@+id/fragment_legend_image"
+        android:layout_width="50dp"
+        android:layout_height="50dp"
+        android:src="@drawable/k"
```

Comment:
So you add that line up next to this one: "xmlns:android="http://schemas.android.com/apk/res/android"

And then in your views, anything you want to just be used as a preview, you replace the "android:" with "tools:". Go ahead and make that change.